README.md

```diff
@@ -31,11 +31,34 @@ Family context expects a process similar to the following to be in place:
 ```
 
 ## Setup
-The easiest way to run this is to do the following commands after you have 
+### Running Locally for Development
+The following steps can be used to run the service in a way that is easy to develop, keeping the database 
+locally and the service easily able to update when changes are made.
+1. Install the libraries needed for the project
+   1. `poetry install`
+   2. Copy the file `.env.sample` to `.env` and fill out the values. If you want to use SQLite, then 
+   delete the database variables
```

Comment:
Which ones are the database variables?

Can I assume that all variable names with "SQL" as prefix, should be deleted at this step?
That is: SQL_ENGINE, SQL_DATABASE, SQL_USER, SQL_PASSWORD, SQL_HOST and SQL_PORT

Reply:
Yes, good point. You're correct. All the SQL variables indicate the database, so if they're absent then it will default to SQLite.